helm-charts/secrets-operator/values.yaml

```diff
@@ -22,17 +22,17 @@ controllerManager:
         memory: 64Mi
   manager:
     args:
-      - --health-probe-bind-address=:8081
-      - --metrics-bind-address=127.0.0.1:8080
-      - --leader-elect
+    - --health-probe-bind-address=:8081
+    - --metrics-bind-address=127.0.0.1:8080
+    - --leader-elect
     containerSecurityContext:
       allowPrivilegeEscalation: false
       capabilities:
         drop:
-          - ALL
+        - ALL
     image:
       repository: infisical/kubernetes-operator
-      tag: v0.8.15
+      tag: <helm-pr-will-update-this-automatically>
```

Comment:
it seems like this only gets updated when the helm PR is generated based on k8 operator patch... What if I just want to add a different configurable parameter in helm? or if I want to add a conditional?